src/app/screens/PeerlyScreens/HomeScreen/index.tsx

```diff
@@ -97,26 +100,33 @@ const HomeScreen = ({navigation}) => {
 
   return (
     <>
-      <View style={styles.header}>
-        <Text style={styles.title}>Peerly</Text>
-        <View style={styles.userScore}>
-          <Text>
-            {profileDetails?.total_points && (
-              <Text style={styles.scoreText}>
-                {profileDetails?.total_points}
-              </Text>
-            )}
-          </Text>
-          <Image
-            source={
-              profileDetails?.profile_image_url
-                ? {uri: profileDetails?.profile_image_url}
-                : require('../../../../assets/images/profile.png')
-            }
-            style={styles.userAvatar}
-          />
+      <Pressable
+        onPress={() =>
+          navigation.navigate(PROFILE_DETAILS, {
+            details: profileDetails,
```

Comment:
We should avoid passing data from one screen to another. 
We should pass an `id` and then fetch the details on the next screen itself.

Reply:
Sure, I will make this change and update.

Reply:
I have done the change in my upcoming PR so you will get updated code in next PR